README.md

```diff
@@ -6,74 +6,52 @@
 
 Echidna is the central piece of software taking care of the new publication workflow at [W3C](http://www.w3.org/). The plan is for Echidna and related sub-projects (see below) to automatise the publication of new specs under [`http://www.w3.org/TR/`](http://www.w3.org/TR/).
 
-## Dependencies
+## Installation
 
-### OS-level dependencies
+To run Echidna, you need to install [Node.js](http://nodejs.org/) first.
+This will install [npm](https://www.npmjs.org/) at the same time, which is required as well.
```

Comment:
You probably don't need this line about npm. As you say, you get it for free with node.